app/src/main/java/org/coepi/android/ui/cen/CENRecyclerAdapter.kt

```diff
@@ -0,0 +1,41 @@
+package org.coepi.android.ui.cen
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import org.coepi.android.R.layout.item_cen
+import org.coepi.android.cen.CEN
+import org.coepi.android.ui.cen.CENRecyclerViewAdapter.ViewHolder
+
+class CENRecyclerViewAdapter : RecyclerView.Adapter<ViewHolder>() {
+    private var items = emptyList<CEN>()
+
+    class ViewHolder(view: View) : RecyclerView.ViewHolder(view) {
+        private val textView = view as TextView
+        fun bind(item: CEN) {
+            textView.text = item.CEN
+        }
+
+    }
+
+    fun setItems(items: List<CEN>) {
+        this.items = items
+        notifyDataSetChanged()
```

Comment:
This is not ideal, but it's just a (probably temporary) debugging UI, so it's fine.